src/Microsoft.ML.TimeSeries/SrCnnBatchAnomalyDetection.cs

```diff
@@ -0,0 +1,778 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Data.DataView;
+using Microsoft.ML.Numeric;
```

Comment:
>Numeric [](start = 19, length = 7)

Is this used?

Reply:
removed.

---
In reply to: [426436746](https://github.com/dotnet/machinelearning/pull/5135#discussion_r426436746) [](ancestors = 426436746)